content/updates/2021-01/2-Manage-your-spell-check-dictionaries.md

```diff
@@ -0,0 +1,18 @@
+---
+pim_announcement_alt_img: "The spell-check dictionaries management feature"
+pim_announcement_audience:
+- EE
+---
+
+# Manage your spell-check dictionaries
```

Comment:
@camillebourjade can you validate the "spell-check dictionaries" wording?
I also saw “Spell Checking Dictionaries” or “Spell dictionaries” on Google.

Reply:
spell-check dictionaries is good. Other forms exist but this one is the most commonly used :-)